### ISSUE TITLE
Invalid arity error in defn with only 1 arity

### ISSUE THREAD

**Issue by NoahTheDuke** [clj-kondo/clj-kondo#1825]
[ To keep development of this project going, consider sponsoring. If you are
already a sponsor, thank you! ]

**version**

```
$ clj-kondo --version
clj-kondo v2022.09.08
```

**macro usage**

[ Is your bug related to macro usage? Consider using a configuration. If not, ignore this section. ]

See [config.md](https://github.com/clj-kondo/clj-kondo/blob/master/doc/config.md#unrecognized-macros) for related configuration options.

**platform**

Linux, Ubuntu 22.04

**editor**

[ If applicable, please specify which editor you are using clj-kondo with and which editor plugin you are using. ]

**problem**

If a `defn` form only has a single arity but the arg vector and body are wrapped in parentheses like there are multiple arities, clj-kondo raises an error about the `defn` call having too few arguments.

**repro**

```
$ echo "(defn a ([b] (inc b)))" | clj-kondo --lint - --config '{}'
<stdin>:1:1: error: clojure.core/defn is called with 2 args but expects 3 or more
linting took 6ms, errors: 1, warnings: 0
```

**config**

As seen above, this happens with default config.

**expected behavior**

No error, lol.

**Comment by NoahTheDuke** [comment]
My apologies, this was a cache issue. The changes between 2022.08.03 and 2022.09.08 to `resources/clj_kondo/impl/cache/built_in/clj/clojure.core.transit.json` (and probably other files) were cached and thus caused the error when running locally. My apologies for any time spent working on this.
